### PR TITLE
test(Card): add test to Card, CardAction, CardPrice, Ribbon and CardF…

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Add test to Card Components**

**How to test it ?**

- Run `yarn test` command in terminal
- Press p to filter by a filename regex pattern
- Write **card** and select **src/components/ChangePlan/Card.test.js**

**Results:**

image

image

### CODE REVIEW

File: src/components/ChangePlan/Card.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 import { Link } from 'react-router-dom';
 
 export const Card = ({ children, className, highlighted, ...rest }) => {
+  // TODO: double negation (!!) should be removed
```
Comment (lucianaDilizio) [inline src/components/ChangePlan/Card.js:7]: 👍

File: src/components/ChangePlan/Card.js [hunk 2]
```diff
@@ -24,6 +25,7 @@ export const CardAction = ({ url, children, ...rest }) => {
   );
 };
 
+// TODO: the variable name 'doNotShowSince' should be an affirmative expression. 'doNotShowSince' is hard to read
```
Comment (lucianaDilizio) [inline src/components/ChangePlan/Card.js:28]: 👍

File: src/components/ChangePlan/Card.js [hunk 3]
```diff
@@ -40,7 +42,7 @@ export const CardPrice = ({ currency, children, doNotShowSince }) => {
   );
 };
 
-export const Ribbon = ({ children, position = 'top-right', content }) => {
+export const Ribbon = ({ position = 'top-right', content }) => {
```
Comment (lucianaDilizio) [inline src/components/ChangePlan/Card.js:45]: 👍

**Comment by andresmoschini** [comment]
👏 👏 👏 

@jcamposmk, maybe we could add a new step in the test to verify that the test coverage has not been reduced or that it is not lower that an arbitrary number. Then, we could increase the number gradually.

**Comment by jcamposmk** [comment]
That would be great!
I don't know if technically it can be done, but it may also be interesting to add a process to verify if there is a test for new developments. It is currently a process that is being done manually. I don't know how feasible it is

**Pull request merged**